Makefile

```diff
@@ -47,6 +47,7 @@ unit-tests:
 	@bash ./scripts/unittests.sh
 
 # test runs ALL the test suites.
-test: systemtests unit-tests
+test:
+	@bash ./scripts/tests.sh
```

Comment:
I just realized systemtests.sh and unittests.sh already check their own status codes and exit, so there's no point in changing this behavior in this PR